libs/execution_context/ebpf_native.c

```diff
@@ -103,7 +103,7 @@ static NPI_PROVIDER_ATTACH_CLIENT_FN _ebpf_native_provider_attach_client_callbac
 static NPI_PROVIDER_DETACH_CLIENT_FN _ebpf_native_provider_detach_client_callback;
 
 static const NPI_PROVIDER_CHARACTERISTICS _ebpf_native_provider_characteristics = {
-    0,
+    1,
```

Comment:
https://learn.microsoft.com/en-us/windows-hardware/drivers/ddi/netioddk/ns-netioddk-_npi_provider_characteristics says this should be set to 0.